RunningOrder/Sprint/SprintDetail.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  SprintDetail.swift
+//  RunningOrder
+//
+//  Created by Lucas Barbero on 05/08/2020.
+//  Copyright © 2020 Worldline. All rights reserved.
+//
+
+import SwiftUI
+
+struct SprintDetail: View {
+
+    var sprint: Sprint
+    @ObservedObject private var viewModel: SprintDetailViewModel
+    @EnvironmentObject var toolbarManager: ToolbarManager
+
+    init(sprint: Sprint) {
+        self.sprint = sprint
+        viewModel = SprintDetailViewModel(sprint: sprint)
+    }
+    var body: some View {
+
+            NavigationView {
+                StoryList(header: "Sprint \(sprint.number) - \(sprint.name)", stories: $viewModel.stories) // TODO Construct/Pass the head in a different way ?
+                    .onAppear {
```

Comment:
why another onAppear, you already have one

Reply:
This one is triggered before the  NavigationView one, isn't it ?